docs/reference/modules/threadpool.asciidoc

```diff
@@ -142,14 +142,23 @@ threadpool:
 [[processors]]
 === Processors setting
 The number of processors is automatically detected, and the thread pool
-settings are automatically set based on it. Sometimes, the number of processors
-are wrongly detected, in such cases, the number of processors can be
-explicitly set using the `processors` setting.
+settings are automatically set based on it. However, the number of
```

Comment:
I get the argument that to do it complete right you need to tweak more things but tbh the only use case I every heard of for `processors` in the wild is to reduce the number of processors in situations where they were shared.  So I think it might be more common than the great addition you made.